api/util/passport.js

```diff
@@ -1,7 +1,7 @@
 const JwtStrategy = require('passport-jwt').Strategy;
 const ExtractJwt = require('passport-jwt').ExtractJwt;
-const User = require('../models/User');
-const { secretKey } = require('../../config/config.json');
+const User = require('../main_endpoints/models/User');
```

Comment:
this is inside main_endpoints, meaning we will still get the below error in docker:
![image](https://user-images.githubusercontent.com/36345325/157809137-bc973a24-586f-49f6-8643-719b4d5cb98b.png)

we have two options:
1. move the user schema to util/
2. make token verification a REST endpoint (`/api/verifyToken`) for other services outside of `main_endpoints` to call on.

1 would be much easier than 2. however the schema would be in an uncommon place. your call.

Reply:
option 2 is the move since it will help later with other token verification in peripheral server